.github/workflows/security-scan.yaml

```
@@ -0,0 +1,159 @@
+        exit-code: '0'
+      continue-on-error: true
+
+    - name: Upload vulnerability scan results to GitHub Security tab
+      uses: github/codeql-action/upload-sarif@v3
+      with:
+        sarif_file: 'trivy-results-${{ matrix.component.name }}.sarif'
+        category: '${{ matrix.component.name }}-vulnerabilities'
+
+    - name: Upload configuration scan results to GitHub Security tab
+      uses: github/codeql-action/upload-sarif@v3
+      if: hashFiles('trivy-config-${{ matrix.component.name }}.sarif') != ''
```

actionlint (1.7.9)

if: condition "hashFiles('trivy-config-${{ matrix.component.name }}.sarif') != ''" is always evaluated to true because extra characters are around ${{ }}

(if-cond)